### PR TITLE
file naming standard

### PR DESCRIPTION
**Pull request by Kitzunu**

### CODE REVIEW

File: docs/Spell_system.md [hunk 1]
```diff
@@ -1,30 +0,0 @@
-##### This article was created and ported from Trinity core's wiki since their system is identical
```
Comment (Kitzunu) [inline docs/Spell_system.md:1]: So no cofusion occurs, this is just a duplicate, and outdated, file of https://github.com/azerothcore/wiki/blob/master/docs/spell_system.md

**Comment by Kitzunu** [comment]
Done @FrancescoBorzi

**Comment by FrancescoBorzi** [comment]
@Kitzunu fine for me as long as all the old links correctly redirect

**Comment by Kitzunu** [comment]
yes

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**